migrations/20220919161227_priority_column_queue.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE mergestat.repo_sync_queue
+ADD COLUMN priority_level NOT NULL INTEGER DEFAULT 0;
```

Comment:
```suggestion
ADD COLUMN priority NOT NULL INTEGER DEFAULT 0;
```
just a suggestion - for me, `priority_level` implies discrete levels like `low` `medium` `high` vs just `priority` doesn't have that same implication, and seems like it fits the numeric approach we're taking here